### ISSUE TITLE
Use with Inline CSS Modules

### ISSUE THREAD

**Issue by jhabdas** [xiaolin/react-image-gallery#109]
This image gallery looks fantastic. I'm working with an _awesome react boilerplate_ called [react-production-starter](https://github.com/jaredpalmer/react-production-starter), which uses Aphrodite for CSS Modules and isomorphic rendering.

Though not ideal, I tried pulling the CSS out of the `dist` directory and adding the `style` and `css` loaders for Webpack as mentioned in https://github.com/xiaolin/react-image-gallery/issues/70 then did an `import './style.css'` from within my `PhotoGallery` component. However, that won't work because [Aphrodite](https://github.com/Khan/aphrodite) will appropriately rename the classes so they do not match the hard-coded class names expected by the component.

Any suggestions for correctly using this library with the CSS Module approach? With 100 forks I'd wager someone has figured out how already, or re-rolled this to use inline styles. Anyone?

**Comment by xiaolin** [comment]
Are you saying `className` defined by a component will be renamed by `Aphrodite`?

e.g. `<div className='image-gallery-bullets'>` becomes `<div className='some_otherclass'>`?

**Comment by jhabdas** [comment]
Good question. I ended up copying the CSS from the NPM module into a local CSS file, which is why the CSS classes are being renamed. Maybe if I `import` directly from the module they won't get renamed... But I do like the idea of being able to use `<tag className={css(styles.whatever)} />` within the image gallery codebase. But I'd prefer not to fork if there's already one out there.

I noticed working with [react-photoswipe](https://github.com/minhtranite/react-photoswipe) I have a similar challenge, so I guess I'm scouting for a photo gallery using a CSS Module approach as opposed to one where I'll need to cajole a Webpack config to make things work.

Either way, thanks for this great project. If you happen across one that uses CSS Modules I'd love to know. Cheers!
